src/pages/createOrderNewFlow.js

```diff
@@ -596,6 +596,7 @@ class CreateOrderPage extends React.Component {
     }
 
     Router.pushRoute(route, { ...params, ...pick(this.props, ['verb', 'tierSlug']) });
+    window.scrollTo(0, 0);
```

Comment:
We usually wait for `pushRoute` to complete before scrolling. Using `await` or `then`.

Reply:
Makes sense! I just pushed a fix for that